Sources/Transition.swift

```diff
@@ -56,6 +56,12 @@ public struct Transition<T: Hashable>: Hashable {
     public var hashValue: Int {
         return (from?.hashValue ?? 0) + (to?.hashValue ?? 0)
     }
+    
+    public var description: String {
+        let f = from != nil ? String(describing: from!) : "unknown"
```

Comment:
Am I correct that this would result in creating a new state `unknown` for `unknown -> A` and `A -> unknown`? Maybe `any -> A` would be better in that case.

Reply:
Yes, you're correct! I hadn't realised nil -> A | A -> nil signified a relative transition.